src/components/ComplaintDetail/ComplaintDetail.js

```diff
@@ -0,0 +1,62 @@
+import './ComplaintDetail.less';
+import React, { useEffect, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  selectDetailActiveCall,
+  selectDetailData,
+  selectDetailError,
+} from '../../reducers/detail/selectors';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { getComplaintDetail } from '../../actions/complaints';
+import iconMap from '../iconMap';
+import Loading from '../Dialogs/Loading';
+import { selectQuerySearch } from '../../reducers/query/selectors';
+import { ComplaintDetailBody } from './ComplaintDetailBody';
+
+export const ComplaintDetail = () => {
+  const location = useLocation();
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const data = useSelector(selectDetailData);
+  const error = useSelector(selectDetailError);
+  const activeCall = useSelector(selectDetailActiveCall);
+  const search = useSelector(selectQuerySearch);
+
+  const isLoading = activeCall !== '';
+  const backUrl = useMemo(() => {
+    const pathName = location.pathname || '';
+    const idx = pathName.indexOf('detail');
+    return pathName.substring(0, idx) + search;
+  }, [location, search]);
+
+  useEffect(() => {
+    dispatch(getComplaintDetail(id));
+  }, [dispatch, id]);
+
+  return (
+    <section className="card-container">
+      <nav className="layout-row">
+        <div className="back-to-search flex-fixed">
+          <Link to={backUrl}>
+            {iconMap.getIcon('left', 'cf-icon-left')}
+            Back to search results
+          </Link>
+        </div>
+        <div className="meaning flex-fixed">
+          <a
+            href="https://www.consumerfinance.gov/complaint/data-use/"
```

Comment:
We possibly should turn these into relative links when they go off to production. I.e. have a search and replace in `build.sh` that turns `https://www.consumerfinance.gov/` into `/`.

Reply:
Good point. 

I was about to fix it in this component like so
```
  const dataUseLink =
    window.location.origin.indexOf('localhost') > -1
      ? 'https://www.consumerfinance.gov/complaint/data-use/'
      : '/complaint/data-use/';
```

I also checked to see where else this occurs. 

Looks like it's in the hero as well.
https://github.com/cfpb/ccdb5-ui/search?q=https%3A%2F%2Fwww.consumerfinance.gov

hero isn't a functional component yet and has a snapshot test that would break in CI so I won't touch it. 

I moved the link into a constant that we can reference when we convert hero and remove the snapshot. 
updated in [1e7ff25f862eb59aba9b54e128d62b9bfebaa18e](https://github.com/cfpb/ccdb5-ui/pull/436/commits/1e7ff25f862eb59aba9b54e128d62b9bfebaa18e)